analyses/molecular-subtyping-LGAT/00-subset-files-for-LGAT.R

```diff
@@ -0,0 +1,44 @@
+# K S Gaonkar 2020
+# Subsets the consensus mutation file to biospecimens where short_histology == LGAT
+
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# get subset folder
+subset_dir<-file.path(root_dir,"analyses","molecular-subtyping-LGAT","lgat-subset")
+
+# clinical file
+clinical<- read_tsv(file.path(root_dir,"data","pbta-histologies.tsv"))
```

Comment:
I would expect to see:

```suggestion
clinical<- readr::read_tsv(file.path(root_dir,"data","pbta-histologies.tsv"))
```